test/OpenTelemetry.Tests/Metrics/MetricPointReclaimTestsBase.cs

```diff
@@ -28,7 +28,7 @@ protected MetricPointReclaimTestsBase(bool emitOverflowAttribute)
         }
 
         this.configuration = new ConfigurationBuilder()
-            .AddInMemoryCollection(this.configurationData)
+            .AddInMemoryCollection(this.configurationData!)
```

Comment:
configurationData is not nullable. Why is this change needed?

Reply:
fixed in b0cbbe775bfa80d9a901c168ecdfeebbb1db0dd0